Add an empty CSV file to avoid a runtime error on referencing
skeinforge/skeinforge_application/profiles/extrusion/Default directory.

--- /dev/null
+++ b/skeinforge_application/profiles/extrusion/Default/skeinforge.csv
@@ -0,0 +1,2 @@
+Format is tab separated skeinforge settings.
+_Name                          	Value
